### PR TITLE
Fixed style issues

### LINKED ISSUE #71: Next button design
The xml for the new design exists but doesn't apply

### PR DESCRIPTION
**Pull request by itaistopel**
image

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,7 +9,6 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```
Comment (svechinsky) [inline app/src/main/AndroidManifest.xml:9]: Why is this commited?

File: app/src/main/res/layout/camera_fragment.xml [hunk 2]
```diff
@@ -47,11 +47,11 @@
 
     <Button
         android:id="@+id/button_startCamera"
-        style="@style/customButton"
```
Comment (svechinsky) [inline app/src/main/res/layout/camera_fragment.xml:50]: Same, should be reverted

File: app/src/main/res/layout/daily_metric_fragment.xml [hunk 3]
```diff
@@ -79,13 +79,13 @@
 
             <Button
                 android:id="@+id/button_metricSubmit"
-                style="@style/customButton"
```
Comment (svechinsky) [inline app/src/main/res/layout/daily_metric_fragment.xml:82]: Same

File: app/src/main/res/layout/fragment_welcome.xml [hunk 4]
```diff
@@ -56,8 +56,8 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginBottom="168dp"
-        android:text="@string/start"
         style="@style/customButton"
+        android:text="@string/start"
```
Comment (svechinsky) [inline app/src/main/res/layout/fragment_welcome.xml:60]: same

File: app/src/main/res/layout/instructions_fragment.xml [hunk 5]
```diff
@@ -58,10 +58,10 @@
 
     <Button
         android:id="@+id/button_instructions"
-        style="@style/customButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/start"
+        style="@style/customButton"
```
Comment (svechinsky) [inline app/src/main/res/layout/instructions_fragment.xml:64]: Same

**Comment by itaistopel** [comment]
@svechinsky 
All of the unwanted changes occurred automatically by the IDE.

Is it necessary to revert those changes? It doesn't affect anything.

**Comment by svechinsky** [comment]
Fair

**Pull request merged**